documentation/src/main/asciidoc/getting_started/example_with_groovy.adoc

```diff
@@ -49,7 +49,13 @@ Groovy Version: 1.6.3 JVM: 1.6.0_14
 If you get a similar result as shown, everything went well.
 
 ==== Installing {brandname}
-Now you should add the {brandname} libraries to the Groovy Platform so you will able to access the API from the Groovy console. Add the infinispan-core.jar and its dependencies to the $USER_HOME/.groovy/lib directory, the jar is located in $INFINISPAN_HOME/modules/core and the dependencies at $INIFINISPAN_HOME/modules/core/lib.
+Now you should add the {brandname} libraries to the Groovy Platform so you will able to access the API from the Groovy console. Add the infinispan-core.jar and its dependencies to the $USER_HOME/.groovy/lib directory, the jar is located in
+ifndef::productized[]
+$INFINISPAN_HOME/modules/core and the dependencies at $INFINISPAN_HOME/modules/core/lib.
```

Comment:
I'm not familiar with the syntax, but is it not possible to have an else clause?

Reply:
I think the short answer is no. https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/_includes/conditional-preprocessor-directives.adoc
I'm definitely on for more elegant approaches if you have suggestions.